packages/sources/celsius-address-list/test/integration/fixtures.ts

```diff
@@ -0,0 +1,31 @@
+import nock from 'nock'
+
+export function mockResponseSuccess(): void {
+  nock('https://test-rpc-url:443', { encodedQueryParams: true })
+    .post('/http', { method: 'eth_chainId', params: [], id: 42, jsonrpc: '2.0' })
+    .reply(200, { jsonrpc: '2.0', result: '0x2a', id: 42 }, [])
+    .post('/http', { method: 'eth_chainId', params: [], id: 43, jsonrpc: '2.0' })
+    .reply(200, { jsonrpc: '2.0', result: '0x2a', id: 43 }, [])
+    .post('/http', {
```

Comment:
Would recommend using the standard Ethereum JSON-RPC URLs. I know you're familiar with the JSON-RPC port 😄 

```suggestion
  nock('https://test-rpc-url:8545', { encodedQueryParams: true })
    .post('/', { method: 'eth_chainId', params: [], id: 42, jsonrpc: '2.0' })
    .reply(200, { jsonrpc: '2.0', result: '0x2a', id: 42 }, [])
    .post('/', { method: 'eth_chainId', params: [], id: 43, jsonrpc: '2.0' })
    .reply(200, { jsonrpc: '2.0', result: '0x2a', id: 43 }, [])
    .post('/', {
```

I suppose this will also have to be updated in the test file as well

Reply:
Lol fixed 👍